extract/extract/extractors/amazon_de.py

```diff
@@ -237,13 +237,14 @@ def _get_image_urls(soup: BeautifulSoup) -> Optional[list[str]]:
             If nothing was found `None` or empty list is returned.
     """
     targets = [
-        soup.find("div", {"id": "altImages"}).find_all("img"),
+        soup.find("div", {"id": "altImages"}),
+        soup.find("div", {"class": "unrolledScrollBox"}),
     ]
 
-    def parse_image_urls(images: list[BeautifulSoup]) -> list[str]:
+    def parse_image_urls(images: BeautifulSoup) -> list[str]:
         image_urls = [
             str(image["src"])
-            for image in images
+            for image in images.find_all("img")
```

Comment:
Maybe all these checks can be moved to a helper fn as it looks a bit overcrowded

Reply:
Hmm, there are quite some checks done. However, these checks are already part of a separate function. 
So, I went with the code as is.